build.gradle.kts

```diff
@@ -68,7 +68,7 @@ tasks {
 
     patchPluginXml {
       sinceBuild.set("222")
-      untilBuild.set("231.*")
+      untilBuild.set("241.*")
```

Comment:
Can we remove specifying this?
Seems like it's not mandatory. 
https://intellij-support.jetbrains.com/hc/en-us/community/posts/360010590059/comments/360002979840

Reply:
Purpose y I added it is to set the sinceBuild. Our plugin base compatibility is java 17 which is supported only since 222.
Will remove it.